sqlmesh/core/loader.py

```diff
@@ -620,6 +577,59 @@ def _load_metrics(self) -> UniqueKeyDict[str, MetricMeta]:
 
         return metrics
 
+    def _load_external_models(
```

Comment:
It looks like the dbt loader [calls this method](https://github.com/TobikoData/sqlmesh/blob/75b7270a5c949ae0c8304f05bad6bf1ec8cc5f83/sqlmesh/dbt/loader.py#L148), so it feels like moving this under `SqlMeshLoader` loader isn't right?